src/models/generated/Bus.jl

```diff
@@ -9,7 +9,7 @@ mutable struct Bus <: Topology
     bustype::Union{Nothing, BusType}  # bus type
```

Comment:
can we add a `Node` abstract type above `Bus`? I think it's confusing to have `Bus` next to `Arch`

Reply:
Maybe we should have:
```
PowerSystems.Topology
├──Arch
└──Node
    ├─Bus
    └─LoadZone
```